### ISSUE TITLE
Renaming households/huishoudens and buildings/gebouwen sectors

### ISSUE THREAD

**Issue by marliekeverweij** [quintel/etmodel#3764]
As @Charlottevm noticed the terms "households" / "huishoudens" & "buildings" / "gebouwen" are actually very confusing terms for many users and I agree! I also noticed this with our clients from Northern-Ireland!
- households sounds like it's about the people living in houses / dwellings, not about the house / dwelling itself
- buildings sounds like  it's about all buildings, including houses / dwellings, so not only utilities

@Charlottevm says that quite a few users first go to buildings when they want to change something to the dwellings....

Proposal for new terms:
- for households
  - EN: 
    1. dwellings
    2. houses
    3. domestic sector
  - NL: woningen
- for buildings sector:
  - EN: 
    1. utility
    2. services
  - NL: utiliteit
  
Please let me us know:

- if you think renaming is necessary
- which new terms you prefer
- if you have better terms in mind!

**Comment by antw** [comment]
> households sounds like it's about the people living in houses / dwellings, not about the house / dwelling itself

I'm not sure I agree. The households sector includes sliders related to the home _and_ the people who live there. There are numerous sliders for per-person demand and behaviour. I think "Households" describes this quite well.

> buildings sounds like it's about all buildings, including houses / dwellings, so not only utilities

Agreed. I think "utility" or "utilities" is the wrong term, though. In English "utilities" is normally used to refer to amenities such as electricity, water, gas, etc.

**Comment by marliekeverweij** [comment]
>I'm not sure I agree. The households sector includes sliders related to the home and the people who live there. There are numerous sliders for per-person demand and behaviour. I think "Households" describes this quite well.

True that. However, there are more sliders about the homes than about the people, so I would prefer to change it.

> Agreed, though I think "utility" or "utilities" is the wrong term. In English "utilities" is normally used to refer to amenities such as electricity, water, gas, etc.

Good to know :) I think "services" better fit here, that's the term that our clients from Northern-Ireland use

**Comment by redekok** [comment]
I agree that clients are often confused by the terms we use in the ETM. Especially "buildings" causes a lot of confusion since this term seems to capture both residences and services. 

As for "households" I don't have a very strong opinion. I think the terms "woningen" and "huishoudens" are both used quite often. Not sure what the best way forward is in English though.

As for "buildings" I prefer to replace it by "services". In Dutch we could also debate whether "utiliteit" or "diensten" is the best term to use.

**Comment by mabijkerk** [comment]
I don't have a strong preference for changing households. Eurostat for example uses 'households' and I agree with @antw that there are a number of sliders that describe behavioural aspects. I'm fine with leaving it as is.

For buildings I agree that the current term can be confusing. I would be in favour of changing it to 'services' for English. In Dutch, I'm not sure that 'utiliteit' would be the most appropriate term, so I have a slight preference for 'diensten'.

**Comment by MartLubben** [comment]
I would prefer changing the 'buildings/gebouwen' sector to services/diensten. I do not have a strong opinion about changing Households/Huishoudens. 

'Dienstverlening' is the specific term used by de Klimaatmonitor. For users I think it will be clear that Diensten and Dienstverlening is equal. 

![image](https://user-images.githubusercontent.com/33415156/127329511-44a8edd1-27dc-4d61-bf26-d23318015b77.png)

**Comment by marliekeverweij** [comment]
Allright, I think we have a shared preference here 🎉 :
- keep "households/huishoudens"
- rename "buildings/gebouwen" to "services/diensten"

I will rename the issue!

Now we have to choose if we will also do the renaming in the back-end.... But I see this as a separate issue.

**Comment by mabijkerk** [comment]
Just to be clear: in my opinion, only the labels in the frontend have to be renamed. I don't think it is necessary to also change the labels in the backend.
